pkgs/by-name/sc/sccmhunter/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "sccmhunter";
+  version = "1.0.6-unstable-2024-10-30";
+  pythonRelaxDeps = true;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "garrettfoster13";
+    repo = "sccmhunter";
+    rev = "7f493d1499fec1a694d949d9de17f25acdb6a0a9";
+    hash = "sha256-rMkm4SDIiPrlNJWlnwD2td7yzkTNlTzZ3KaiKK5TOfg=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    cmd2
+    cryptography
+    impacket
+    ldap3
+    pandas
+    pyasn1
+    pyasn1-modules
+    requests
+    requests-kerberos
```

Comment:
Why add `requests-kerberos`? I don't find it in dependencies upstream.

https://github.com/garrettfoster13/sccmhunter/blob/7f493d1499fec1a694d949d9de17f25acdb6a0a9/requirements.txt

Reply:
Yes, during my testing / building of the packages, it was a required package that was not listed within `requirements.txt`.

But, it seems that this was only by mistake (https://github.com/garrettfoster13/sccmhunter/issues/75) and the dependency has since then been removed (https://github.com/garrettfoster13/sccmhunter/commit/4c6669e0f2fbaf9ed9f96edb8eac0bc0f41cf28b).

Should I bump the commit rev to a newer commit where `requests-kerberos` is not included anymore?

Reply:
It seems the using rev currently does not have it anymore. It's ok to just remove this dependency here.

https://github.com/garrettfoster13/sccmhunter/blob/7f493d1499fec1a694d949d9de17f25acdb6a0a9/lib/attacks/admin.py